--- origsrc/kdebindings-4.5.4/generator/CMakeLists.txt	2010-11-02 07:51:11.000000000 -0500
+++ src/kdebindings-4.5.4/generator/CMakeLists.txt	2011-01-03 00:08:42.941419100 -0600
@@ -32,7 +32,8 @@ add_executable(smokegen ${generator_SRC}
 target_link_libraries(smokegen ${QT_QTCORE_LIBRARY} ${QT_QTXML_LIBRARY} cppparser)
 set_target_properties(smokegen PROPERTIES COMPILE_DEFINITIONS __GENERATOR_BUILDING ENABLE_EXPORTS TRUE)
 
-install(TARGETS smokegen DESTINATION ${CMAKE_INSTALL_PREFIX}/bin)
+install(TARGETS smokegen RUNTIME DESTINATION ${CMAKE_INSTALL_PREFIX}/bin
+                         ARCHIVE DESTINATION ${CMAKE_INSTALL_PREFIX}/lib)
 install(FILES generatorenvironment.h  generator_export.h  generatorpreprocessor.h  generatorvisitor.h  name_compiler.h  options.h  type_compiler.h  type.h DESTINATION ${CMAKE_INSTALL_PREFIX}/include/smokegen)
 
 add_subdirectory(generators)
--- origsrc/kdebindings-4.5.4/generator/parser/CMakeLists.txt	2010-11-02 07:51:11.000000000 -0500
+++ src/kdebindings-4.5.4/generator/parser/CMakeLists.txt	2011-01-03 00:09:21.379886100 -0600
@@ -38,4 +38,5 @@ add_definitions(-D__CPPPARSER_BUILDING)
 add_library(cppparser SHARED ${cppparser_LIB_SRCS})
 target_link_libraries(cppparser ${QT_QTCORE_LIBRARY})
 
-install(TARGETS cppparser LIBRARY DESTINATION ${LIB_INSTALL_DIR})
+install(TARGETS cppparser LIBRARY DESTINATION ${LIB_INSTALL_DIR}
+                          RUNTIME DESTINATION bin)
