--- origsrc/kdebindings-4.6.4/generator/smokeapi/main.cpp	2011-01-18 15:23:54.000000000 -0600
+++ src/kdebindings-4.6.4/generator/smokeapi/main.cpp	2011-06-21 23:19:27.752313800 -0500
@@ -38,8 +38,8 @@ static QRegExp targetPattern;
 
 static Smoke* 
 loadSmokeModule(QString moduleName) {
-    QFileInfo file(QString("libsmoke") + moduleName);
-    QLibrary lib(file.filePath());
+    QFileInfo file(QString("smoke") + moduleName);
+    QLibrary lib(file.filePath(), 3);
 
     QString init_name = "init_" + moduleName + "_Smoke";
     InitSmokeFn init = (InitSmokeFn) lib.resolve(init_name.toLatin1());

